src/vs/editor/contrib/snippet/browser/snippetVariables.ts

```diff
@@ -292,6 +293,14 @@ export class TimeBasedVariableResolver implements VariableResolver {
 			return TimeBasedVariableResolver.monthNamesShort[this._date.getMonth()];
 		} else if (name === 'CURRENT_SECONDS_UNIX') {
 			return String(Math.floor(this._date.getTime() / 1000));
+		} else if (name === 'CURRENT_TIMEZONE_OFFSET') {
+			const rawTimeOffset = this._date.getTimezoneOffset();
+			const sign = rawTimeOffset < 0 ? '-' : '+';
```

Comment:
```suggestion
			const sign = rawTimeOffset > 0 ? '-' : '+';
```

Sign should be inverted.  ISO 8601 is positive *east* of GMT while `getTimezoneOffset` is positive *west* of GMT.  Zero should always be positive.

Reply:
Examples: `Asia/Kolkata` should be `+05:30`.  `America/Phoenix` should be `-07:00`.

Reply:
Thanks for spotting the error, I've fixed it.